Fix RegionGeom's ASDF roundtrip for DS9-format regions (#6801)
gammapy/gammapy: docs/release-notes/6801.bug.rst

> Fix ``RegionGeom`` ASDF roundtrip for DS9-format regions where ``meta`` and ``visual`` were not cleared consistently on write and read.

Reviewer: ```suggestion
Fix `~gammapy.maps.RegionGeom` ASDF roundtrip for DS9-formatted regions to clear ``meta`` and ``visual`` on write.
```
